lib/command-log.js

```diff
@@ -0,0 +1,22 @@
+'use babel'
```

Comment:
also if we're switching to using es6 imports etc...can we apply the same style to the rest of the codebase as well. just want to have it match one way or antoher